src/tox/session/commands/show_config.py

```diff
@@ -29,7 +29,10 @@ def show_config(config):
     content = StringIO()
     parser.write(content)
     value = content.getvalue().rstrip()
-    reporter.verbosity0(value)
+    # We are not using the reporter because showconfig result should not be
+    # affected by logging level as its stdout output *is* expected to
+    # be machine parseable.
```

Comment:
That explanation should then also be in the other places where this promise is made. If we have a little dedicated section in the docs about the commands that are machine readable they could all have a comment like `# log level independent output - intended for machines to read (see docs/...)`

Reply:
I was not able to identify any suitable place inside the docs for mentioning this. If you really want one new section please mention exactly where it should be added inside the docs.

I am afraid that the effort to add this to the docs would be wasted because it would not be disoverable by the users that will want to use this. All of these will use the --help instead, and that is why I mentioned it there, so
you do not endup using a feature without knowing how to use it in a reliable way.

Reply:
I also had a quick look in the docs now. I also wouldn't really know where to put it as it is. A new section for tool builders that build on top of tox, seems overkill also, so I guess having it in the command line help help, is the best place currently.